com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Scripts/DebugUIHandlerEnumHistory.cs

```diff
@@ -9,17 +9,18 @@ namespace UnityEngine.Rendering.UI
     public class DebugUIHandlerEnumHistory : DebugUIHandlerEnumField
     {
         Text[] historyValues;
-        const float xDecal = 60f;
+        const float k_XOffset = 230f;
```

Comment:
should it be in its own DebugPanel Styles class?

Reply:
Perhaps. I don't feel like refactoring this code too much (especially the runtime part) because I think we should replace this UGUI code with UITK.